labelbox/data/serialization/labelbox_v1/objects.py

```diff
@@ -259,33 +259,40 @@ def from_common(cls, text_entity: TextEntity,
                    **extra)
 
 
+class LBV1DocumentRectangle(LBV1Rectangle):
```

Comment:
Starting a thread here in response to your reply, so we can better keep track of the conversation.

It doesn't seem like good practice to create a new Data type specific entity here, as it doesn't follow the current pattern. These geometric classes only hold data about the dimensions of the annotations, and not dimensions of the data types like Document. If we follow this pattern, we'd be creating a Document class for every type of annotation. I'd suggest either putting them in the base class `LBV1ObjectBase` with default values set to None (which seems like the current pattern, since `LBV1ObjectBase` contains optional fields like `color` and `instanceURI`), although this isn't ideal either, since we'd again, be storing data type specific attributes in a annotation related object. Another solution could be to create a new superclass of `LBV1LabelAnnotations` for documents specifically, which would follow the same pattern as video, and has its own `to_common` and `from_common` methods. @msokoloff1, would love your input here as well.

Reply:
Also, was there a reason why only `to_common` was implemented? Does `from_common` not need any code changes?

Reply:
Sounds good. One option I was looking at earlier was adding it under the [feature class here](https://github.com/Labelbox/labelbox-python/blob/73e2336fd06a4627c1054088295265dfe7e53768/labelbox/data/serialization/labelbox_v1/feature.py#L9). It houses `keyframe` but thought that it didn't necessarily make sense, and agree it would make more sense under `LBV1ObjectBase` 

No reason why only `to_common` was implemented. This was just up for discussion so I wanted a better agreement on where it should land for design. I'll implement `from_common` as well

Reply:
Actually, what is convenient is by adding it with `to_common` in the extras field, `from_common` doesn't need any update work at all 😄